.travis.yml

```diff
@@ -15,10 +15,10 @@ android:
     - '.+'
 
 script:
-  - ./gradlew clean assemble check -Dpre-dex=false --stacktrace
+ - ./gradlew clean :json:assemble :json:test :app:assembleDebug :app:check -Dpre-dex=false --stacktrace
```

Comment:
This makes sure we are assembling and testing json(it does not have `assembleDebug` or `assembleTest`).

Then for the Android project, run `assembleDebug` and run all `check`s.